core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -313,6 +303,33 @@ final class ShuffleBlockFetcherIterator(
   }
 }
 
+// Helper class that ensures a ManagedBuffer is released on InputStream.close()
+private class WrappedInputStream(
+              delegate: InputStream, buf: ManagedBuffer, var currentResult: FetchResult)
```

Comment:
I looked at the style guide and used the recommendation for `def` parameters (which are aligned with the definition). I didn't see that I should use the Scala Style Guide; I'll fix this. Thanks for point it out.